bindings/matrix-sdk-ffi/src/ruma.rs

```diff
@@ -487,25 +487,25 @@ impl TryFrom<RumaMessageType> for MessageType {
 }
 
 #[derive(Clone, uniffi::Enum)]
-pub enum NotifyType {
+pub enum RtcNotificationType {
     Ring,
     Notify,
```

Comment:
Should it be renamed `Notification` to align with Ruma?

Reply:
right, done https://github.com/matrix-org/matrix-rust-sdk/pull/5668/commits/9d53bc1a801a21cbd527ec693c9e36217d98373b